libs/ui/src/usbcontroller_button.tsx

```diff
@@ -31,7 +32,8 @@ export function UsbControllerButton({
 }: Props): JSX.Element | null {
   const variant: ButtonVariant = primary ? 'primary' : 'regular';
 
-  if (usbDriveStatus === 'mounted') {
+  const { status } = usbDriveStatus;
```

Comment:
We might want this button to disable immediately after pressing it to avoid double presses (and maybe show a spinner or something while the action is in progress). When I removed the "ejecting" state, I imagined we would do that by waiting on the eject mutation to succeed.

Reply:
Good point, will look into reviving the "ejecting" state via passing in some query state.

Reply:
I revived the "Ejecting..." text by using the `isLoading` flag on the mutation, which I think prevents the obvious issue of double ejecting.

Reply:
Will `usbDriveStatus` still be `'mounted'` while `isLoading` is true? If so, I don't think we'll actually hit the disabled case. Did you test this? Or am I just reading the code wrong?

Reply:
Oops 🤦 . Not reading the code wrong. I did test it briefly, but was focused on making sure the mutation status was resetting properly and didn't actually confirm I saw the "Ejecting..." status.